db/migrate/20250816213210_add_audit_timestamps_to_collection_preferences.rb

```diff
@@ -0,0 +1,20 @@
+class AddAuditTimestampsToCollectionPreferences < ActiveRecord::Migration[7.1]
+  uses_departure! if Rails.env.staging? || Rails.env.production?
+
+  def change
+    change_table :collection_preferences, bulk: true do |t|
+      t.datetime :unrevealed_updated_at
+      t.datetime :anonymous_updated_at
+    end
+
+    reversible do |dir|
+      dir.up do
+        execute <<~SQL.squish
+          UPDATE collection_preferences
+          SET unrevealed_updated_at = created_at,
+              anonymous_updated_at  = created_at
+        SQL
+      end
+    end
```

Comment:
same as my comment on `set_initial_audit_timestamps` though here with the added note that I think this would confuse us in the future, since it would be set to created_at although the setting was changed at some point before this PR was merged